solvebio/cli/data.py

```diff
@@ -505,6 +506,113 @@ def _download(full_path, local_folder_path, dry_run=False):
             file_.full_path, local_folder_path, file_.filename))
 
 
+def sync(args):
+    """
+    Given a folder or file, download all the files contained
+    within it (not recursive)
+    """
+    return _sync(args.full_path, args.local_path, dry_run=args.dry_run)
+
+
+def _get_remote_directory_recursive(full_path, local_folder_path, remote_files=[]):
+    """Queries a SolveBio Vault path and recursively finds all files.
+
+    full_path (str): SolveBio path where data will be downloaded from.
+    local_folder_path (str): Local folder path where data will be downloaded to.
+    remote_files (list): List used to store queried files in recursive search.
+    """
+    objects = Object.all(glob=full_path, limit=1000)
+    for object_ in objects:
+        # TODO: Skip dataset objects for now?
```

Comment:
I think that is fine, just leave a comment and we can also log/print that a dataset (log the ID and the full path) is being skipped